### PR TITLE
docs: remove D-Bus and polkit note, it is not true.

### PR DESCRIPTION
**Pull request by kayrus**
/cc @jonboulle 
This wiki page is outdated http://www.freedesktop.org/wiki/Software/systemd/dbus/
Polkit rule was already implemented in this PR: https://github.com/coreos/coreos-overlay/pull/1579

### CODE REVIEW

**Comment by jonboulle** [comment]
oh really? interesting. but can we change this to a note about how packagers/deployers can do this then?

**Comment by jonboulle** [comment]
And worth pinging upstream systemd about that doc, I don't know where they keep the source..

**Comment by kayrus** [comment]
https://github.com/systemd/systemd/issues/2562

**Comment by kayrus** [comment]
https://github.com/systemd/systemd/issues/1042

**Comment by jonboulle** [comment]
perhaps https://github.com/coreos/fleet/pull/1427#issuecomment-180441196 and then we can move forward with this?

**Comment by kayrus** [comment]
@jonboulle updated doc. added more security notes.

**Comment by kayrus** [comment]
I've also added new fleetd option into docs.

File: Documentation/deployment-and-configuration.md [hunk 1]
```diff
@@ -2,15 +2,51 @@
...
-Each `fleetd` daemon must be configured to talk to the same [etcd cluster][etcd]. By default, the `fleetd` daemon will connect to either http://127.0.0.1:4001 or http://127.0.0.1:2379, depending on which endpoint responds. Refer to the configuration documentation below for customization help.
+Each `fleetd` daemon must be configured to talk to the same [etcd cluster][etcd]. By default, the `fleetd` daemon will connect to either http://127.0.0.1:2379 or http://127.0.0.1:4001, depending on which endpoint responds. Refer to the configuration documentation below for customization help.
 
-`fleet` requires etcd be of version 0.3.0+.
+`fleet` requires etcd be of version 0.3.0+ but it is recommended to use etcd 2.0.0+ which supports [TLS authentication][etcd-security].
 
-[etcd]: https://coreos.com/docs/cluster-management/setup/getting-started-with-etcd
+### TLS Authentication
+
+If you have etcd cluster with enabled TLS authentication you have to configure fleet to use TLS keypair. This example will should you how to do that:
```
Comment (jonboulle) [inline Documentation/deployment-and-configuration.md]: If your etcd cluster has TLS authentication enabled, you will need to configure fleet to use an appropriate TLS keypair. The example shows below how to achieve this:

File: Documentation/architecture.md [hunk 2]
```diff
@@ -50,18 +50,28 @@ A UnitState object represents the state of a Unit in the fleet engine. A UnitSta
...
 
-Current releases of fleet don't currently perform any authentication or authorization for submitted units. This means that any client that can access your etcd cluster can potentially run arbitrary code on many of your machines very easily.
+Current releases of fleet don't currently perform any authentication or authorization for submitted units. This means that any client that can access your etcd cluster can potentially run arbitrary code on many of your machines very easily, thus it is strongly recommended to enable [TLS authentication][fleet-tls] on the etcd side and set proper file permissions to the keypair on the host.
 
 ## Securing etcd
 
 You should avoid public access to etcd and instead run fleet [from your local laptop][using-the-client] with the `--tunnel` flag to run commands over an SSH tunnel. You can alias this flag for easier usage: `alias fleetctl=fleetctl --tunnel 10.10.10.10` - or use the environment variable `FLEETCTL_TUNNEL`.
 
-## Other Notes
+## Securing fleet
```
Comment (jonboulle) [inline Documentation/architecture.md]: s/fleet/fleetd/

File: Documentation/architecture.md [hunk 3]
```diff
@@ -50,18 +50,28 @@ A UnitState object represents the state of a Unit in the fleet engine. A UnitSta
...
 
-Since it interacts directly with systemd over D-Bus, the fleetd daemon must be run with elevated privileges (i.e. as root) in order to perform operations like starting and stopping services. From the [systemd D-Bus documentation][systemd-dbus]:
+It is also recommended to run fleetd under separate `fleet` user and group, and set listening unix socket permissions to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. While fleet daemon uses [D-Bus][d-bus] it is necessary to create a [`polkit(8)`][polkit] rule to allow fleetd to communicate with systemd:
```
Comment (jonboulle) [inline Documentation/architecture.md]: set the permissions of the fleetd API's listening Unix socket

Comment (jonboulle) [inline Documentation/architecture.md]: s/While fleet daemon/Since the fleet daemon uses D-Bus to communicate with systemd/

**Comment by jonboulle** [comment]
A few minor tweaks and needs a rebase. LGTM after that.

**Comment by kayrus** [comment]
Done. @jonboulle final review?

**Comment by jonboulle** [comment]
lgtm

**Comment by jonboulle** [comment]
thanks

**Pull request merged**